sqlx-postgres/src/message/row_description.rs

```diff
@@ -74,7 +74,7 @@ impl BackendMessage for RowDescription {
 
             fields.push(Field {
                 name,
-                relation_id: if relation_id == 0 {
+                relation_id: if relation_id.0 == 0 {
```

Comment:
```suggestion
                relation_id: if relation_id == 0 {
```

Reply:
Alternatively you can continue to wrap it when assigning `relation_id` for clarity and do `== Oid(0)` here.